examples/javascript-developer/executor.mjs

```diff
@@ -17,5 +17,10 @@ $.tool.executor.runToolExecutor({
         workspacePath: process.env.WORKSPACE,
       }),
     }),
+    $.tool.getUserInput({
+      execute: $.tool.executeGetUserInput({
+        workspacePath: process.env.WORKSPACE,
+      }),
+    }),
```

Comment:
`executor.mjs` is just required for tools that run in the docker container. The user input does not need to be in hte container - it can be directly added to `runDeveloperAgent.ts`.